.github/workflows/pull_request.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Xcode Build
       run: |
         set -o pipefail
-        xcodebuild build -workspace COVIDWatch.xcworkspace -scheme 'COVIDWatch iOS' CODE_SIGNING_ALLOWED=NO | xcpretty
+        xcodebuild build -workspace COVIDWatch.xcworkspace -scheme 'covidwatch-ios-dev' CODE_SIGNING_ALLOWED=NO | xcpretty
```

Comment:
Should we add the prod one to the [fastfile](https://github.com/covid19risk/covidwatch-ios/blob/dev/fastlane/Fastfile)?

Reply:
Yes, good call.